tests/features/common/kie-kieserver-common.feature

```diff
@@ -458,3 +458,8 @@ Feature: Kie Server common features
       | MQ_SERVICE_PREFIX_MAPPING | AMQPREFIX |
     Then container log should contain Configuring external JMS integration, removing /opt/eap/standalone/deployments/ROOT.war/META-INF/kie-server-jms.xml
      And file /opt/eap/standalone/deployments/ROOT.war/META-INF/kie-server-jms.xml should not exist
+
+  Scenario: RHDM-1096 - Check that optaplanner thread pool queue size property is not set without env.
+    When container is started with env
+      | variable        | value  |
+    Then container log should not contain -Dorg.optaplanner.server.ext.thread.pool.queue.size=
```

Comment:
In addition to a test that checks that the sys  prop is not set when the env var is not set, there should be a test that checks that the sys prop is set to the expected value when the env var is set.

Reply:
Done!